### PR TITLE
Contradicting comments removed

### PR DESCRIPTION
**Pull request by ANSHUMAN87**

### CODE REVIEW

File: tensorflow/lite/allocation.cc [hunk 1]
```diff
@@ -32,15 +32,11 @@ namespace tflite {
 FileCopyAllocation::FileCopyAllocation(const char* filename,
                                        ErrorReporter* error_reporter)
     : Allocation(error_reporter) {
-  // Obtain the file size, using an alternative method that is does not
```
Comment (talumbau) [inline tensorflow/lite/allocation.cc]: Thanks for this change! I think better would be to put an accurate comment. Something like:

"Obtain the file size using fstat, or report an error if that fails."

**Comment by ANSHUMAN87** [comment]
@talumbau : Your comment is handled now, Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by gbaned** [comment]
@talumbau Can you please take a look on this PR? Thanks!

**Review by rthadur — APPROVED** [review]

**Pull request merged**